autouri/__init__.py

```diff
@@ -119,27 +119,20 @@ def main():
 
     elif args.action == 'cp':
         u_src = AutoURI(src)
-        sep = AutoURI(target).__class__.get_path_sep()
-        if target.endswith(sep):
-            type_ = 'dir'
-            target = sep.join([target.rstrip(sep), u_src.basename])
-            print(target)
-        else:
-            type_ = 'file'
         _, flag = u_src.cp(target, make_md5_file=args.make_md5_file)
 
         if flag == 0:
-            logger.info('Copying from file {s} to {type} {t} done'.format(
-                s=src, type=type_, t=target))
+            logger.info('Copying from file {s} to {t} done'.format(
```

Comment:
For backwards compatibility reasons the logging uses by default the old % style formatting. Using { style (str.format) in general seems to work, but may cause unexpected problems. Fortunately configuring this is pretty straightforward.
Instead of using `logging.basicConfig(level=logging.INFO, format='%(asctime)s %(levelname)s %(message)s')` to set your format, you would omit the `format` parameter when you set the `logging.basicConfig` and instead create a `Formatter` object and set the format and style there `formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s', style='{')` and add the formatter to your logger with `logger.setFormatter(formatter)`.

Reply:
Will fix it as you suggested.

Reply:
Even though I use `style='{'` that does not allow something like the following, which is explained [here](https://stackoverflow.com/questions/13131400/logging-variable-data-with-new-format-string):
logger.info('hello {target}', target='world')

I will revert it and keep using `%` style formatting since I always pass to the logger a string that is already done with formatting.

Reply:
Yeah, I have always just used the old % style to make sure. That definitely works.